lib/msal-node/test/test_kit/ManagedIdentityTestUtils.ts

```diff
@@ -96,7 +96,7 @@ export class ManagedIdentityNetworkClient implements INetworkModule {
     ): Promise<NetworkResponse<T>> {
         return new Promise<NetworkResponse<T>>((resolve, _reject) => {
             resolve({
-                status: 200,
+                status: HttpStatus.SUCCESS_RANGE_START,
```

Comment:
I much preferred 200. Easy to read.

Reply:
I agree... but don't we want to avoid "magic numbers", no matter how obvious 200 is?